cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/ZillabaseDockerCommand.java

```diff
@@ -41,10 +62,59 @@ protected final void invoke()
             .build();
 
         DockerClient client = DockerClientImpl.getInstance(config, httpClient);
+        final ZillabaseConfig zillabaseConfig = readZillabaseConfig();
 
-        invoke(client);
+        invoke(client, zillabaseConfig);
```

Comment:
Does every command need `zillabase` `config`?
If not then perhaps we can keep the method `protected` instead of `private` and call from commands that need it?